### PR TITLE
Update staking and rates documentation

### PR DESCRIPTION
**Pull request by dwightjl**

### CODE REVIEW

**Comment by biddleb** [comment]
Line 53 in the realized vol section should read "If you have questions or would like to request an enhancement to Realized Volatility feeds, contact us using [this form](https://forms.gle/Jc1MGY5ry1ceYH1X6)."  Currently, it says "ETH Staking APR". 

Similarly, line 41 under the ETH Staking APR section should say "ETH Staking APR" where it currently says Realized Volatility.

**Review by biddleb — APPROVED** [review]

File: src/pages/data-feeds/rates-feeds/index.mdx [hunk 1]
```diff
@@ -32,14 +34,20 @@ See the [Rate and Volatility Feed Addresses](/data-feeds/rates-feeds/addresses)
...
 
 ## Realized volatility
 
-Realized volatility measures how much an asset price has changed in the past. This value is expressed as a percent of the asset price. The more an asset price changes over time, the higher the realized volatility is for that asset. Prices are measured at regular intervals to track the change in an asset's price. Realized volatility is not the same as implied volatility, which is a measure of the market’s expectation about future volatility.
+Realized volatility measures how much an asset price has moved around in the past. This value is expressed as a percent of the asset price. The more an asset price moves up or down over time, the higher the realized volatility is for that asset. Please note that realized volatility is not the same as implied volatility, which is a measure of the market’s expectation about future volatility typically derived from options markets.
```
Comment (ritika-b) [inline src/pages/data-feeds/rates-feeds/index.mdx:45]:
```suggestion
Realized volatility measures asset price movement over a specific time interval. This value is expressed as a percent of the asset price. The more an asset price moves up or down over time, the higher the realized volatility is for that asset. Please note that realized volatility is not the same as implied volatility, which measures the market’s expectation about future volatility typically derived from options markets.
```

File: src/pages/data-feeds/rates-feeds/index.mdx [hunk 2]
```diff
@@ -32,14 +34,20 @@ See the [Rate and Volatility Feed Addresses](/data-feeds/rates-feeds/addresses)
...
+
+Each data feed reflects the volatility of an asset over a specific rolling window of time. For example, there are data feeds that provide volatility data for the last 24 hours, 7 days, and 30 days of time. You can compare the data across these windows to infer whether the volatility of an asset is trending up or down. For example, if realized volatility for the 24-hour window is higher than the 7-day window, this suggests volatility might be increasing.
```
Comment (ritika-b) [inline src/pages/data-feeds/rates-feeds/index.mdx:47]:
```suggestion
Each data feed reflects the volatility of an asset over a specific rolling window of time. For example, some data feeds provide volatility data for the last 24 hours, 7 days, and 30 days of time. You can compare the data across these windows to infer whether the volatility of an asset is trending up or down. For example, if realized volatility for the 24-hour window is higher than the 7-day window, volatility might increase.
```

File: src/pages/data-feeds/rates-feeds/index.mdx [hunk 3]
```diff
@@ -32,14 +34,20 @@ See the [Rate and Volatility Feed Addresses](/data-feeds/rates-feeds/addresses)
...
+
+The same high-quality data providers used in Chailink’s price feeds sample price data every 10 minutes to refresh volatility estimates, and on-chain values are updated when the feed heartbeat or deviation threshold are met.
```
Comment (ritika-b) [inline src/pages/data-feeds/rates-feeds/index.mdx:49]:
```suggestion
The same high-quality data providers used in Chailink’s price feeds sample price data every 10 minutes to refresh volatility estimates. On-chain values are updated when the feed heartbeat or deviation threshold is met.
```

**Review by ritika-b — CHANGES_REQUESTED** [review]

**Review by ritika-b — APPROVED** [review]

**Pull request merged**